README.md

```diff
@@ -136,6 +136,12 @@ Specifies a function to call on a footnote popover that is being activated (afte
 
 Default: `undefined`
 
+### `deactivateCallback`
```

Comment:
Should be `dismissCallback` for consistency.

Reply:
indeed, sounds better!